javascript/challenges/fizzBuzzTree/fizz-buzz-tree.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { Node } = require('../../data-structures/tree/tree');
+const { BinaryTree } = require('../../data-structures/tree/tree');
+
+function fizzBuzzTree(tree) {
+  if (!tree.root) {
+    throw new RangeError('Cannot operate on an empty tree!');
+  }
+
+  function _fizzBuzzCheck(root) {
+    if (!root) {
+      return;
+    }
+
+    if (root.value % 3 === 0 && root.value % 5 === 0) {
+      root.value = 'FizzBuzz';
+    } else if (root.value % 5 === 0) {
+      root.value = 'Buzz';
+    } else if (root.value % 3 === 0) {
+      root.value = 'Fizz';
+    } else {
+      root.value.toString();
```

Comment:
I think that this is what is causing that bug you talked about in your submission

Reply:
Fixed up this bug and got all non-divisible numbers converting to strings correctly and all tests passing.